### PR TITLE
Package-level README cleanup

### PR DESCRIPTION
**Pull request by gigabo**
This is what shows up on [npmjs.com](https://www.npmjs.com/package/react-server).  Needs to be a good pitch.

### CODE REVIEW

File: packages/react-server/README.md [hunk 1]
```diff
@@ -1,169 +1,33 @@
...
+based on the [Flux pattern](https://facebook.github.io/flux/docs/overview.html).
+Read more on [our dev blog](https://www.redfin.com/devblog/2015/09/thoughts-on-transitioning-to-universal-javascript.html).
 
-Back when we started this project, we came up with a great name -- `triton` -- and we started using it internally. And then it took us forever to get around to making the source code public. And then [Joyent](https://www.joyent.com) released a product called `triton` and stole our thunder, so we had to go with `react-server` instead (a terrible, terrible tragedy, we know).
+## Getting started
 
-[![Dependency Status](https://david-dm.org/redfin/react-server.svg)](https://david-dm.org/redfin/react-server)
-[![devDependency Status](https://david-dm.org/redfin/react-server/dev-status.svg)](https://david-dm.org/redfin/react-server#info=devDependencies)
+For now it's easiest to get started with
+[`react-server-cli`](https://github.com/redfin/react-server/tree/master/packages/react-server-cli).
```
Comment (doug-wade) [inline packages/react-server/README.md]: I would link to npmjs.org/package/react-server-cli ; the easiest way to get started is not to learn about how monorepos work.

Comment (gigabo) [inline packages/react-server/README.md]: Good call.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -15,6 +15,18 @@ You could:
 Head over to [react-server-test-pages](/packages/react-server-test-pages) and
 check out the README to get a test server set up.
 
+## Testing
+
+Yeah!  Do it!
```
Comment (doug-wade) [inline CONTRIBUTING.md]: I would move the sentence at ll 15 to after "Do it!" -- it explains how to do smoke testing.  As is, its a little imposing

**Comment by doug-wade** [comment]
coupla nits, otherwise lgtm.  Thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (gigabo) [inline CONTRIBUTING.md]: Hmm... `react-server-test-pages` doesn't actually have any automated tests ([yet](https://github.com/redfin/react-server/issues/147)).

But I guess that makes sense.

**Comment by doug-wade** [comment]
👍

**Pull request merged**